system-addon/content-src/components/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,84 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {FormattedMessage} = require("react-intl");
+const {actionTypes, actionCreators: ac} = require("common/Actions.jsm");
+
+/**
+ * Confirmation Dialog component.
+ * One primary action button, one cancel button.
+ *
+ * Content displayed is controlled by `data` prop the component receives.
+ * Example:
+ * data: {
+ *   // Any sort of data needed to be passed around by actions.
+ *   payload: site.url,
+ *   // Primary button SendToMain action.
+ *   action: "DELETE_HISTORY_URL",
+ *   // Primary button USerEvent action.
+ *   userEvent: "DELETE",
+ *   // Array of locale ids to display.
+ *   message_body: ["confirm_history_delete", "confirm_history_delete_notice"],
```

Comment:
@k88hudson The design has the two texts separated. As two different paragraphs. Any way to achieve this in an a11y way?
[link to design](https://mozilla.invisionapp.com/share/49C81C6Z7#/screens/238928525_Photon_-_Confirm_Dialog)